engines/router/missionctl/log/resultlog/proto/turing/TuringResultLog.proto

```diff
@@ -16,11 +16,16 @@ message Request {
 }
 
 message Response {
-    // The JSON response from a Turing component (Enricher / Experiment Engine / Router / Ensembler), UTF-8-encoded.
-    string response = 1;
+    // The JSON response body from a Turing component (Enricher / Experiment Engine / Router / Ensembler),
+    // UTF-8-encoded.
+    string body = 1;
```

Comment:
Renamed `response` as `body` because the addition of the new field `header` would create some ambiguity as to what `response` would really mean.

Reply:
That's a valid point. But unfortunately, this will be a breaking change, not for the proto decoding itself, but rather, the downstream workflows that depend on it. For example, in most internal use cases, the Kafka messages are forwarded to BQ and the field name change would result in a change in the table as well.

Similarly, our own BQ logger uses the schema generated from this proto. This means, the next time users update their routers, they have to carry out a schema migration on the existing table as the only thing we handle automatically is addition of new columns ([ref](https://github.com/gojek/turing/blob/main/engines/router/missionctl/log/resultlog/bigquery.go#L238)).

We can look into a schema migration if required (which we have done once in the past) but it would require the users to take actions to migrate their existing data / scripts as well. So we should avoid it unless required.

Considering all of the above, I think `header`, `response` is not too outlandish and we can live with it.